app/src/main/java/org/mozilla/fenix/library/historymetadata/interactor/HistoryMetadataGroupInteractor.kt

```diff
@@ -67,14 +72,18 @@ class DefaultHistoryMetadataGroupInteractor(
     }
 
     override fun onDeleteMenuItem(items: Set<History.Metadata>) {
-        // no-op
+        controller.handleDeleteItem(items)
     }
 
     override fun onDeleteAllMenuItem() {
-        // no-op
+        controller.handleDeleteAll()
     }
```

Comment:
These are technically in the History three-dot menu but it's related and the functionality-less buttons are already in `main`, so I figured I'd roll it into this

Reply:
See interace descriptions above ☝️